README.md

```diff
@@ -57,9 +57,48 @@ This driver supports only one topology key:
 that represents availability by zone.
 
 ## Kubernetes User Guide
-
 ### Install Driver
 
+#### Preflight (Ubuntu like)
+1. You need to install golang
+```
+sudo apt update && sudo apt install golang
+```
+2. Set path for go
+You can read [here](https://github.com/golang/go/wiki/SettingGOPATH) more or quick run this steps
+Set path to /home/$USER/go
+```
+export GOPATH=$HOME/go
```

Comment:
again, maybe this is something specific that we can link to official golang instructions for